examples/kft-v2/scripts/trl_training.py

```
@@ -0,0 +1,790 @@
+        if test_dataset is not None:
+            test_columns = list(test_dataset.features.keys())
+            test_columns.remove('text') if 'text' in test_columns else None
+            test_dataset = test_dataset.map(template_dataset, remove_columns=test_columns)
+
+        return train_dataset, test_dataset
+
+    def get_training_parameters():
+        """Get training parameters."""
+        checkpoint_dir = Path(os.getenv('CHECKPOINT_URI', '/workspace/checkpoints'))
+        checkpoint_enabled = os.getenv('CHECKPOINT_ENABLED', 'false').lower() == 'true'
+        checkpoint_interval = os.getenv('CHECKPOINT_INTERVAL', '30s')
```

Ruff (0.12.2)

Local variable `checkpoint_interval` is assigned to but never used

Remove assignment to unused variable `checkpoint_interval`

(F841)